receipt_places/receipt_places/adapter.py

```
@@ -0,0 +1,260 @@
+    OpeningHoursPeriod,
+    Photo,
+    Place,
+    PlusCode,
+    Viewport,
+)
+from receipt_places.types_v1 import PlaceV1
+
+logger = logging.getLogger(__name__)
+
+
+def adapt_v1_location_to_legacy(location: Optional["receipt_places.types_v1.Location"]) -> Optional[LatLng]:
```

Ruff (0.14.8)

Undefined name `receipt_places`

(F821)